File: Clinica.UI/src/app/models/atendimento.ts
import { Especialista } from "./especialista";
import { Paciente } from "./paciente";

export interface Atendimento {
  id: number;
  data: string;
  dataRetorno: string;
  hora: string;
  duracao: string;
  observacao: string;
  valor: number;
  status: number;
}

File: Clinica.API/Controllers/models/Atendimento.cs
using System.Collections.Generic;

namespace Clinica.API.Controllers.models
{
    public class Atendimento
    {
        public int Id { get; set; }
        public string data { get; set; }
        public string dataRetorno { get; set; }
        public string hora { get; set; }
        public string duracao { get; set; }
        public string observacao { get; set; }
        public long valor { get; set; }
        public int status { get; set; }
        public Paciente Paciente { get; set; }
    }
}